app/models/content_item.rb

```diff
@@ -171,12 +171,20 @@ def router_rendering_app
     rendering_app || "government-frontend"
   end
 
-  def viewable_by?(user_uid)
-    authorised_user_uids.empty? || authorised_user_uids.include?(user_uid)
+  def viewable_by_user_id?(user_id)
+    auth_user_ids.include?(user_id)
   end
 
-  def includes_auth_bypass_id?(auth_bypass_id)
-    auth_bypass_ids.include?(auth_bypass_id)
+  def viewable_by_org_id?(org_id)
+    auth_org_ids.include?(org_id)
+  end
+
+  def viewable_by_bypass_id?(bypass_id)
+    auth_bypass_ids.include?(bypass_id)
+  end
+
+  def access_limited?
+    auth_bypass_ids.present? || auth_user_ids.present? || auth_org_ids.present?
```

Comment:
For this method I'd probably go with `auth_user_ids.any? || auth_organisation_ids.any?` and leave out auth_bypass_ids since that doesn't apply a limit (it's a bit confusing it's in that hash really)

Reply:
Changed